app/views/travel_requests/show.html.erb

```diff
@@ -60,6 +60,7 @@
       <card size="full-width">
         <card-content>
           <data-table
+            class="estimates"
```

Comment:
@sdellis this one also should be combined with line 67

Reply:
Removed the overriding class